jbpm-test-coverage/src/test/java/org/jbpm/test/functional/subprocess/SubprocessesTest.java

```diff
@@ -59,7 +59,7 @@
     private static final String HELLO_WORLD_PROCESS = "org/jbpm/test/functional/common/HelloWorldProcess1.bpmn";
     private static final String HELLO_WORLD_PROCESS_ID = "org.jbpm.test.functional.common.HelloWorldProcess1";
 
-    private static final boolean JAVA8 = System.getProperty("java.version").contains("1.8");
+    private static final boolean JAVA8 = System.getProperty("java.version").contains("1.8") || System.getProperty("java.version").startsWith("9.");
```

Comment:
I would rename it to JAVA8_OR_9 then.

Reply:
it's initial version to see if changes done for jdk9 also work on jdk 8 :) once they do we can do additional cleanup etc

Reply:
renamed